source/Coop.Core/Client/Services/Settlements/Messages/SettlementOwnershipChangeRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+
+namespace Coop.Core.Client.Services.Settlements.Messages
+{
+    /// <summary>
+    /// Event for settlement ownership change request sent from client to server
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public class SettlementOwnershipChangeRequest : ICommand
```

Comment:
Please create an integration test for this message. See https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/development/source/Coop.IntegrationTests/MobileParties/PartyBehaviorTest.cs